README.md

```diff
@@ -1,73 +1,100 @@
-# Role Name
+# crypto_policies
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This Ansible role manages system-wide crypto policies.
+
+This concept is well adopted since Red Hat Enterprise Linux 8 and in Fedora.
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+The system-wide crypto policies are implemented and tested on RHEL 8/CentOS 8
+and Fedora.
+
+The role might work elsewhere, but might need enabling in [vars/](vars/).
```

Comment:
It is not clear what "enabling in vars" means/what the user needs to do.

Reply:
Does the following sound more understandable?

> The role might work elsewhere, but will require adding OS definitions in [vars/](vars/).